cpu/cc26xx_cc13xx/vectors.c

```diff
@@ -158,12 +158,14 @@ ccfg_regs_t cc26xx_cc13xx_ccfg = {
                         & CCFG_SET_BITS(SET_MODE_CONF_1_TCXO_MAX_START,
                                         CCFG_MODE_CONF_1_TCXO_MAX_START_s,
                                         CCFG_MODE_CONF_1_TCXO_MAX_START_m)
-#else
-                        0
-#endif /* CPU_VARIANT_X2 */
                         & CCFG_SET_BITS(SET_MODE_CONF_1_ALT_DCDC_VMIN,
                                         CCFG_MODE_CONF_1_ALT_DCDC_VMIN_s,
                                         CCFG_MODE_CONF_1_ALT_DCDC_VMIN_m)
+#else
+                          CCFG_SET_BITS(SET_MODE_CONF_1_ALT_DCDC_VMIN,
```

Comment:
No & here?

Reply:
The asigment is direct this way,

```c
    .MODE_CONF_1 =  CCFG_SET_BITS(SET_MODE_CONF_1_ALT_DCDC_VMIN,
                   ....
```

I could add the `#ifdef` block only for the `&` but seemed very weird:

```c
#ifdef CPU_VARIANT_X2
                        /* only x2 CPUs allow TCXO settings */
                          CCFG_SET_BITS(SET_MODE_CONF_1_TCXO_TYPE,
                                        CCFG_MODE_CONF_1_TCXO_TYPE_s,
                                        CCFG_MODE_CONF_1_TCXO_TYPE_m)
                        & CCFG_SET_BITS(SET_MODE_CONF_1_TCXO_MAX_START,
                                        CCFG_MODE_CONF_1_TCXO_MAX_START_s,
                                        CCFG_MODE_CONF_1_TCXO_MAX_START_m)
                        &
#else
                          CCFG_SET_BITS(SET_MODE_CONF_1_ALT_DCDC_VMIN,
                                        CCFG_MODE_CONF_1_ALT_DCDC_VMIN_s,
                                        CCFG_MODE_CONF_1_ALT_DCDC_VMIN_m)
#endif /* CPU_VARIANT_X2 */
```

Reply:
Ok